MonoGameProj/MonoGameProj/Constants/DirectionConstants.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MonoGameProj.Constants
+{
+    public enum DirectionConstants
```

Comment:
May have to refactor this later. DSurely this will hold ActionConstants like SHOOTING as well?